pkg/syncer/admissionhandler/validatepvc.go

```diff
@@ -145,7 +145,7 @@ func getPVReclaimPolicyForPVC(ctx context.Context, pvc corev1.PersistentVolumeCl
 	kubeClient, err := k8s.NewClient(ctx)
 	if err != nil {
 		return result, logger.LogNewErrorf(log, "failed to get kube client with error: %v. "+
-			"Stopping getting reclaim policy for PVC, %s/%s", err, pvc.Namespace, pvc.Name)
+			"Stopping getting reclaim policy for PVC, %s/%s", pvc.Namespace, err, pvc.Name)
```

Comment:
We don't need change here, it should be on line 154

Reply:
done.